ethereum_history_api/circuits/lib/src/misc/uint256.nr

```diff
@@ -0,0 +1,54 @@
+use dep::std::ops::Add;
+
+global pow128 = 340282366920938463463374607431768211456; // 2^128
+
+struct U256 {
+    high: U128,
+    low: U128,
+}
+
+impl U256 {
+    fn new(high: U128, low: U128) -> Self {
+        Self { high, low }
+    }
+
+    fn from_u64s(u64s: [u64; 4]) -> Self {
```

Comment:
When do we plan to use this? It looks more like serde implementation. I think - this should live in a trait